tox.ini

```diff
@@ -304,13 +305,14 @@ passenv=
 commands =
 # run only essential tests related to the tracing client
     tracer: pytest {posargs} tests/tracer
-    profile: python -m tests.profiling.run pytest --capture=no --verbose --benchmark-disable {posargs} tests/profiling
+    # profile: python -m tests.profiling.run pytest --capture=no --verbose --benchmark-disable {posargs} tests/profiling
```

Comment:
```suggestion
```

I don't really mind leaving it in either but wanted to make sure it was intentional

Reply:
Yes I've intentionally left that there but perhaps would be best to put a comment that explains why we are excluding profile from coverage. WDYT?

Reply:
comment would be nice